tools/buildtools/ambertools/mmpbsa_mmgbsa.xml

```diff
@@ -0,0 +1,238 @@
+<tool id="mmpbsa_mmgbsa" name="mmpbsa mmgbsa" version="@VERSION@">
+  <description>- estimate ligand binding affinities </description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements">
+      <expand macro="jinja_requirement"></expand>
```

Comment:
you don't need this ...
you can simply include: `<requirement type="package">jinja2</requirement>`